src/style.css

```diff
@@ -1,12 +1,145 @@
 body {
   background-color: whitesmoke;
   font-family: 'Open Sans', 'Lucida Grande', tahoma, verdana, arial, sans-serif !important;
+  min-height: 100vh;
+  position: relative;
+  margin: 0 !important;
+  padding: 0 !important;
+}
+
+header {
+  width: 100%;
+  height: 7em;
+  display: flex;
+  align-items: center;
+}
+
+header > div {
+  width: 1550px;
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  align-items: center;
+  gap: 8em;
+}
+
+.fd-number {
+  font-weight: bold;
+  color: black;
+}
+
+.fd-number span {
+  font-weight: bold;
+  color: red;
+}
+
+.fd-categories {
+  font-weight: bolder;
+  text-transform: uppercase;
+  color: darkgray;
+  white-space: nowrap;
+}
+
+.bubbles {
+  display: inline-block;
+  font-family: arial;
+  position: relative;
+}
+
+.bubbles h1 {
+  position: relative;
+  font-family: 'Luckiest Guy', cursive;
+  color: rgb(77, 70, 70);
+  z-index: 2;
+  white-space: nowrap;
+}
+
+.individual-bubble {
+  position: absolute;
+  border-radius: 100%;
+  bottom: 10px;
+  background-color: rgb(93, 79, 79);
+  z-index: 1;
 }
 
 main {
   display: flex;
   width: 100%;
-  height: 100vh;
   justify-content: center;
   align-items: center;
+  padding-bottom: 5em;
+}
+
+main .grid-container {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  padding: 10px;
+}
+
+main .grid-item {
+  border: none;
+  padding: 20px;
+  font-size: 30px;
+  text-align: center;
+}
+
+.food {
+  font-size: small;
+}
+
+.food > div {
+  margin-bottom: 2em;
+}
+
+.food-name {
+  text-align: left;
+}
+
+.food-name .like {
+  float: right;
+}
+
+.food-name .like:hover {
+  transform: scale(120%);
+}
+
+.likes {
+  text-align: right;
+}
+
+.grid-container input[type="button"] {
+  width: 100%;
```

Comment:
[OPTIONAL] There is an observable duplicate of `width: 100%` on your codebase as seen here, and on lines 128, 134, 143, 67... It would be better to give all HTML tags common class names (say; full-width) and style them together, separately. This would make the code look DRY, making it more readable and reusable.

**Kindly, apply this to anywhere in your code where there are visible duplicates of properties**